nss-day-cohort-48/giffygram-gopher
# New Giffy Post bug fix

#### Changes Made

1. Modified file `PostEntry.js` to include `boolean value true or false` functionality to show or hide the entry form.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout  `anm-newPostBugFix`
    ```
2. Refresh browser

3. Test app functionality.
    > Entry form should be hidden on page load, only show when Have a gif to post is clicked
    > When submitting new giffy post, entry form should stay and input fields are cleared

4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
